e2e/src/usecases/HandleDeepLinkSend.js

```diff
@@ -99,6 +100,13 @@ export default HandleDeepLinkSend = () => {
       await openDeepLink(PAY_URL)
       await expect(element(by.id('SendAmount'))).not.toBeVisible()
     })
+
+    it('should navigate to another screen correctly', async () => {
+      await openDeepLink(deepLinks.navigateToSend)
+      await waitFor(element(by.id('RecipientPicker')))
+        .toBeVisible()
+        .withTimeout(10 * 1000)
+    })
```

Comment:
Great adding a test! 
But now I'm surprised this works, as we don't to allow deep links to arbitrary screens, for security reasons. Except when it's from ourselves (push notifications).

https://github.com/valora-inc/wallet/blob/f3b008bb6297ec679419ad8604449e474029632a/src/app/saga.test.ts#L119-L126

I wonder what code path it takes and if CleverTap may be handling it, causing `secureOrigin` to be true: https://github.com/valora-inc/wallet/blob/f3b008bb6297ec679419ad8604449e474029632a/src/app/App.tsx#L90